src/amo/components/AddonMeta/styles.scss

```diff
@@ -42,3 +42,23 @@
   display: flex;
   justify-content: center;
 }
+
+.AddonMeta .MetadataCard-content,
+.AddonMeta .MetadataCard-title {
+  a,
+  a:link,
+  a:visited {
+    color: $grey-50;
```

Comment:
I think the number should be black, like before, instead of gray. This will match the mocks.

Your patch:

<img width="224" alt="screenshot 2018-08-14 10 38 40" src="https://user-images.githubusercontent.com/55398/44102080-7f50fd4a-9fae-11e8-815a-3eb52438e0a5.png">


The mocks:

<img width="284" alt="screenshot 2018-08-14 10 39 04" src="https://user-images.githubusercontent.com/55398/44102095-85fec366-9fae-11e8-8f6c-dd0ec74d8a69.png">


I also have questions about how to make these look more like links but let's save that for a follow-up. I like how the links turn blue when you hover over them.